lightning/src/routing/gossip.rs

```diff
@@ -1575,7 +1660,7 @@ impl<L: Deref> NetworkGraph<L> where L::Target: Logger {
 	/// pruning occur for updates which are at least two weeks old, which we implement here.
 	///
 	/// This function takes the current unix time as an argument. For users with the `std` feature
-	/// enabled, [`NetworkGraph::remove_stale_channels`] may be preferable.
+	/// enabled, [`NetworkGraph::remove_stale_channels_and_tracking`] may be preferable.
 	pub fn remove_stale_channels_with_time(&self, current_time_unix: u64) {
```

Comment:
We don't remove stale tracking in this method, only in `remove_stale_channels_and_tracking`. This is because we decided to just hold instants in the tracking maps and not u64 timestamps.

Reply:
Could you link a comment where you decided that? It slightly affects #1671 as well.

Reply:
In that issue, I need to track the time channels were removed in `remove_stale_channels_with_time`. For that, I'd rather have `removed_channels` map use timestamps rather than `Instant`.

Reply:
Sorry, didn't catch these comments, @naumenkogs!
Matt suggested using `Instant`s [here](https://github.com/lightningdevkit/rust-lightning/pull/1649#discussion_r952002682), since we're not worrying about persisting these maps.

Reply:
I guess I might as well change this in #1671 anyway.